packages/blockchain-wallet-v4-frontend/src/components/ExpansionPanel/ExpansionPanel.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { Icon } from '@blockchain-com/constellation'
+import { IconChevronDown, IconChevronUp } from '@blockchain-com/icons'
+
+import { Button } from 'blockchain-info-components'
+import { Flex } from 'components/Flex'
+
+import { ExpansionPanelComponent } from './types'
+
+export const ExpansionPanel: ExpansionPanelComponent = ({ children }) => {
+  const [isToggled, handleToggle] = useState(false)
+
+  return (
+    <div>
+      {isToggled ? <div>{children}</div> : ''}
+      <Button
+        nature='white-blue'
+        small
+        style={{ margin: '8px auto 0px' }}
```

Comment:
Could you remove this style prop and wrap the button in a `<Padding></Padding>` component? Here's some instructions https://blockchain.github.io/blockchain-wallet-v4-frontend/?path=/story/structural-padding-intro--page

Reply:
Thank you Dylan!! Could you re check it please?